### ISSUE TITLE
Doesn't work without Turbolinks

### ISSUE THREAD

**Issue by morgoth** [basecamp/google_sign_in#7]
After PR https://github.com/basecamp/google_sign_in/pull/6 it stopped working with simple Rails application which doesn't use Turbolinks because now form is submitted as `xhr` with javascript content type, thus controller would need to return js response, so simple `redirect_to` as stated in readme doesn't work anymore.

Adding option `local: true` to https://github.com/basecamp/google_sign_in/blob/6b34248df356ffb5afa25aac929e49768ade6f01/lib/google_sign_in/helper.rb#L28 fixes the problem.

Can we find a solution to not make this simple gem turbolniks dependent?

Note: It also doesn't work when one has `Rails.application.config.turbolinks.auto_include = false`

**Comment by dhh** [comment]
I think, begrudgingly, maybe we go back to recommending that you have to use `turbolinks: false` when linking to a page using the google sign-in. @javan?

**Comment by morgoth** [comment]
Is there anything I can do to help resolve the issue?
@dhh What do you mean by `turbolinks: false` option? I found only https://github.com/turbolinks/turbolinks#disabling-turbolinks-on-specific-links but I guess it's only for GET links?

**Comment by javan** [comment]
This issue isn't technically about Turbolinks. You'd have the same problem with any form in your app using `form_with` (`<form data-remote="true">`) and `redirect_to`.

> thus controller would need to return js response

You can do that pretty easily in your app:
```ruby
render js: "window.location = #{url_for(user).to_json}"
```
Which could be wrapped up into helper:
```ruby
def js_redirect_to(options)
  render js: "window.location = #{url_for(options).to_json}"
end
```

**Comment by morgoth** [comment]
Yes, it's not difficult to do it this way, but it doesn't work "out of the box" with default Rails app setup. Also, readme now is not correct about the controller example.

Should we go with updating readme file or do you think it can be simplified to use `local: true` option or any other workaround?

**Comment by morgoth** [comment]
What about adding option `:local` to `google_sign_in` helper that defaults to false (or true if we want to have defaults simple), so one can overwrite it with `google_sign_in(url: session_path, local: true/false)`?

**Comment by javan** [comment]
Mind giving that ^ a try?

**Comment by morgoth** [comment]
It works. Thank you. Can we have a release maybe? :)

**Comment by javan** [comment]
That's @dhh territory

**Comment by dhh** [comment]
Released 0.1.3 👍
